zebedee-cms/src/main/java/com/github/onsdigital/zebedee/logging/v2LoggingHelpers.java

```diff
@@ -0,0 +1,41 @@
+package com.github.onsdigital.zebedee.logging;
+
+import com.github.onsdigital.zebedee.model.Collection;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.function.BiFunction;
+
+public class v2LoggingHelpers {
+
+    private static final String BLOCKING_PATH = "blockingPath";
+    private static final String BLOCKING_COLLECTION = "blockingCollection";
+    private static final String TARGET_PATH = "targetPath";
+    private static final String TARGET_COLLECTION = "targetCollection";
+
+    private static BiFunction<String, String, String> COLLECTION_CONTENT_PATH = (collectioName, uri) -> {
```

Comment:
logic looks fine but I don't think this needs to be a function. You can just make this a static method with the same logic. I typically use this approach if I want to be able to pass in the function (usually in unit tests or if I am writing a reusable function).